### PR TITLE
Minting

### PR DESCRIPTION
**Pull request by jeapostrophe**
- no Algorand

### CODE REVIEW

File: hs/Makefile [hunk 1]
```diff
@@ -10,11 +10,30 @@ install-mo:
 mo:
 	@which mo 1>/dev/null || (echo 'FAILURE: Please call `make install-mo` first' && exit 1)
 
-package.yaml: package.mo.yaml ../VERSION mo
+package.yaml: package.mo.yaml ../VERSION
```
Comment (DanBurton) [inline hs/Makefile:13]: Why the change on this line?

Comment (jeapostrophe) [reply hs/Makefile:13]: Because `mo` is phony, it thinks that this always needs to be rebuilt
